test/samples/indentation.clj

```diff
@@ -60,6 +60,8 @@
 (clojure.core/filter even?
                      (range 1 10))
 
+(#'filter even?
```

Comment:
I the indentation OK if it was just `'filter`? Also it might be good to change the example to make more sense in the context of quoting being used.

Reply:
I am not sure how `'filter` should indent. Specifically, the reason `var` is quoted is because it implements `'IFn` (ie. it can be called) while a symbol can't be. Let me know if you agree with this. On the others, happy to change the name.

Reply:
Yeah, I'm OK with this.